{{cookiecutter.project_name}}/server/settings/components/common.py

```diff
@@ -54,6 +54,7 @@
 
     # Django:
     'django.middleware.security.SecurityMiddleware',
+    'django_feature_policy.FeaturePolicyMiddleware',  # django-feature-policy
```

Comment:
Is it a proper place to add this middleware? Just asking

Reply:
Yes, this is a correct place after reading of ["Installation"](https://github.com/adamchainz/django-feature-policy#installation) section.
We should put it after `SecurityMiddleware`